webpack.config.js

```diff
@@ -166,11 +168,25 @@ function generatePlugins({ isProduction, isPrerendering, scrivitoOrigin }) {
       },
     ]),
     new MiniCssExtractPlugin({
-      filename: "[name].css",
+      filename: "assets/[name].[contenthash].css",
+    }),
+    new HtmlWebpackPlugin({
+      filename: "catch_all_index.html",
+      template: "catch_all_index.html",
+      inject: false,
```

Comment:
* [ ] Our source files have self-closing `link` tags. According to the docs,  `xhtml: true` renders the link tags as self-closing (XHTML compliant).
I don't know it that only holds true for injected link tags, or all link tags, but for consistency we should turn it on.

Reply:
I did not use `inject: false` to get around the self closing tags issue. I did disable it, because `HtmlWebpackPlugin` would otherwise include _all_ webpack entry assets to the HTML file (e.g. `index.js`, `tracking.js` and `scrivito_extensions.js`).

With https://github.com/Scrivito/scrivito_example_app_js/pull/283/files#diff-149066e1413b1174eae4005fff6923c7R15 you can control much better where specific asset URLs are placed.

Reply:
There's a misunderstanding here: My comment is not about `inject: false`, it is about the options object to the plugin in general.

1. In our HTML templates, we use self-closing link tags.
2. In the options, we tell the plugin to *not* use self-closing link tags.

This is an inconsistent setup.

Reply:
We checked that the setting does *not* change existing `link`s, so it's good enough.